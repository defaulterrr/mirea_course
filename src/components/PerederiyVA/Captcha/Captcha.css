.perederiy_va_captcha {
    display: inline-block;
}