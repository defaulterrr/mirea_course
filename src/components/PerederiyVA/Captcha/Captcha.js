import React from 'react'
import ReCaptcha from 'react-google-recaptcha'
import './Captcha.css'

// const Captcha = () => {
//     return (
//         <div className="perederiy_va_captcha">
//             <ReCaptcha
//                 sitekey="6LdRIfQUAAAAAE-IILR878RvfBGsxTbWJCsUpQfA"
//                 onChange={console.log("captcha has changed")}
//             />
//         </div>
        
//     )
// }

class Captcha extends React.Component {
    render = () => {
        return (
                    <div className="perederiy_va_captcha">
                        <ReCaptcha
                            sitekey="6LdRIfQUAAAAAE-IILR878RvfBGsxTbWJCsUpQfA"
                            onChange={console.log("captcha has changed")}
                        />
                    </div>
                    
                )
    }
}

export default Captcha