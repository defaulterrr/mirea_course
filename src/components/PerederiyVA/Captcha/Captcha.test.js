import React from 'react'
import {create} from 'react-test-renderer'

import Captcha from './Captcha'

describe("[RENDER] Captcha - Snapshot test", () => {
    test("Matches the snapshot", ()=> {
        const captcha = create(<Captcha/>)
        expect(captcha.toJSON()).toMatchSnapshot()
    })
})

