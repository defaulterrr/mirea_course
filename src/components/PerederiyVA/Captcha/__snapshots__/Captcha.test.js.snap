// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[RENDER] Captcha - Snapshot test Matches the snapshot 1`] = `
<div
  className="perederiy_va_captcha"
>
  <div />
</div>
`;
